paasta_tools/cli/cmds/spark_run.py

```diff
@@ -614,6 +625,9 @@ def _parse_user_spark_args(
     if enable_compact_bin_packing:
         user_spark_opts["spark.kubernetes.executor.podTemplateFile"] = pod_template_path
 
+    if enable_spark_dra:
```

Comment:
I considered this case. I think the behaviour in this case should be to enable dra because `--enable-dra` is a more global config than `spark.dynamicAllocation.enabled` so it should override `spark.dynamicAllocation.enabled=false`. Please let me know your thoughts

Reply:
Error out in case of config false and --enable-dra provided

Reply:
Makes sure to suggest the solution in the error.